extensions/wikia/CategorySelect/CategorySelectController.class.php

```diff
@@ -8,7 +8,7 @@
  * @author Kyle Florence <kflorence@wikia-inc.com>
  */
 
-class CategorySelectController extends WikiaController {
+class CategorySelectController extends WikiaService {
```

Comment:
It is - here: https://github.com/Wikia/app/blob/85205157ef0f5f697438128d9b0b9aa7982989f4/extensions/wikia/CategorySelect/js/CategorySelect.view.js#L83

We should only limit accessibility of the vulnerable method.